### PR TITLE
Add absolute link to frontail bootstrap css

### PR DESCRIPTION
**Pull request by ecdye**
Signed-off-by: Ethan Dye <mrtops03@gmail.com>

### CODE REVIEW

File: includes/openhab2-override.conf [hunk 1]
```diff
@@ -1,5 +1,3 @@
 [Unit]
 Description=openHAB2 instance, reachable at http://%H:8080
-Wants=zram-config.service
-Wants=frontail.service
-
+Wants=zram-config.service frontail.service
```
Comment (mstormi) [inline includes/openhab2-override.conf:3]: are you sure this works? I looked around the Inet when I've added it but did not find a confirmation so went the safe route (it's also better to have separate lines should we decide to dynamically generate/change this one day via script)

Comment (ecdye) [reply includes/openhab2-override.conf:3]: I looked at the MAN page and it said it works.

Comment (mstormi) [reply includes/openhab2-override.conf:3]: Please validate first.

Comment (mstormi) [reply includes/openhab2-override.conf:3]: you mean you did validate?

Comment (ecdye) [reply includes/openhab2-override.conf:3]: Not yet. I will this afternoon.

Comment (ecdye) [reply includes/openhab2-override.conf:3]: It works, no issues for me.

**Pull request merged**